src/OrleansServiceBus/Providers/Streams/EventHub/EventDataExtensions.cs

```diff
@@ -31,14 +33,14 @@ public static string GetStreamNamespaceProperty(this EventData eventData)
         public static byte[] SerializeProperties(this IDictionary<string, object> properties)
         {
             var writeStream = new BinaryTokenStreamWriter();
-            SerializationManager.Serialize(properties, writeStream);
+            SerializationManager.Serialize(properties.Where(kvp => !SkipProperties.Contains(kvp.Key)).ToList(), writeStream);
```

Comment:
should you do ToDictionary() instead of ToList()... I don't really care that much, but it's probably cleaner/more logically correct to have an actual dictionary, instead of a list that you then need to convert to dictionary after deserialization